crates/subspace-networking/src/node_runner.rs

```diff
@@ -29,6 +29,8 @@ use tracing::{debug, error, trace, warn};
 
 // Defines a threshold for starting new connection attempts to known peers.
 const CONNECTED_PEERS_THRESHOLD: usize = 10;
+// Defines a protocol name specific for the relay server
+const RELAY_HOP_PROTOCOL_NAME: &[u8; 31] = b"/libp2p/circuit/relay/0.2.0/hop";
```

Comment:
Such things are typically stored in slices, not arrays:
```suggestion
const RELAY_HOP_PROTOCOL_NAME: &[u8] = b"/libp2p/circuit/relay/0.2.0/hop";
```

Reply:
Copied from the source. No problem though.